docs/src/components/ComponentVariableTable.tsx

```diff
@@ -6,7 +6,9 @@ function extractClasses(themeObject) {
   const themeKeys = Object.keys(themeObject);
   let classNames = [];
   themeKeys.forEach((key) => {
-    if (themeObject[key]?.name) {
+    // 'value' is a special attribute, only design tokens will have 'value'
+    // however, there could be a 'name' in part of the object
+    if (themeObject[key]?.value) {
```

Comment:
Updated this logic because there is a 'name' part of the fileuploader theme object.